From d1a7bb7922f4f2cbaf75c8069f9d065abf3dadc1 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Wed, 6 Nov 2024 10:42:22 +0100
Subject: [PATCH] arm64: dts: st: fix mdf compatible on stm32mp231

Use st,stm32mp23-mdf compatible for STM32 MDF in stm32mp231.dtsi.

Change-Id: I9eaef650a4d33a773384455e28c332aaf8d4a364
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/416726
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp231.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp231.dtsi b/arch/arm64/boot/dts/st/stm32mp231.dtsi
index 08b816a70ed6..c4e6600c746d 100644
--- a/arch/arm64/boot/dts/st/stm32mp231.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp231.dtsi
@@ -1578,7 +1578,7 @@ channel@18 {
 			};
 
 			mdf1: mdf@404d0000 {
-				compatible = "st,stm32mp25-mdf";
+				compatible = "st,stm32mp23-mdf";
 				ranges = <0 0x404d0000 0x1000>;
 				reg = <0x404d0000 0x8>, <0x404d0ff0 0x10>;
 				#address-cells = <1>;
-- 
2.50.1

